src/main/java/co/cask/gcp/gcs/actions/GCSBucketCreate.java

```diff
@@ -126,20 +123,22 @@ public void run(ActionContext context) throws Exception {
    * Config for the plugin.
    */
   public final class Config extends GCPConfig {
-
-    @Name("bucket")
-    @Description("Name of the bucket.")
-    @Macro
-    public String bucket;
-
     @Name("paths")
-    @Description("List of objects to be created within a bucket")
+    @Description("Comma separated list of objects to be created.")
     @Macro
-    public String paths;
+    private String paths;
 
     @Name("failIfExists")
     @Description("Fail if path exists.")
     @Macro
-    public String failIfExists;
+    private String failIfExists;
+
+    public List<String> getPaths() {
+      return Arrays.stream(paths.split(",")).map(String::trim).collect(Collectors.toList());
+    }
+
+    public boolean getFailIfExists() {
+      return failIfExists.equalsIgnoreCase("yes");
```

Comment:
looks like there's a mismatch between the widget and this, where the widget has True and False but this... expects 'yes'. This is really weird, don't know why this plugin decided to be different. Let's change it from a String into a Boolean.